### ISSUE TITLE
Add zygisk support to Edxposed

### ISSUE THREAD

**Issue by phantomboy0** [ElderDrivers/EdXposed#916]
Add zygisk support to Edxposed
Mgisk don't support Riru with zygisk enable

**Comment by CodeSpoof** [comment]
Duplicate of #915

**Comment by kb5rir** [comment]
Lsposed works with zygisk and passes safetynet. I like ED better LS. I hope yall get it working with zygisk

**Comment by LucasBS1** [comment]
LSPosed doesn't work on android 8, so I'm basically screwed. Only EDXposed works here.

**Comment by phantomboy0** [comment]
> LSPosed doesn't work on android 8, so I'm basically screwed. Only EDXposed works here.

Yes indeed , many people still uses androids blow the Android 8.1 like myself
I asked LS posed for Android 8.0 support and they responded that they can't do that
Only hope is that Edxpose add the zygisk version

**Comment by treny34** [comment]
Hello, I hope EdXposed support for Zygisk will come soon.

**Comment by waconde** [comment]
me too, really need to support Zygisk
